src/main/java/org/usfirst/frc/team3786/robot/subsystems/ShooterSubsystem.java

```diff
@@ -0,0 +1,32 @@
+package org.usfirst.frc.team3786.robot.subsystems;
```

Comment:
Could you add documentation to the wiki on the specifics of the actuators and/or sensors for this Subsystem if they are already known?  If these are still in flux, could you add an issue to confer with the relevant hardware designers and to add the aforementioned documentation to the wiki?